test/integration/consul-container/test/gateways/http_route_test.go

```diff
@@ -540,8 +550,7 @@ func TestHTTPRouteParentRefChange(t *testing.T) {
 			{
 				Services: []api.HTTPService{
 					{
-						Name:      service.GetServiceName(),
-						Namespace: namespace,
```

Comment:
Was this causing issues? It looks like the gateway, route and service will all reside in the same namespace so this would be optional; however, I wouldn't expect it to cause a problem if set.

Reply:
so in the ENT tests I'm putting the gateway in a different namespace than the route/service (though talking about it, it would probably be better to have these tests always exercise the flows in the same namespace and add additional tests in ent for resources in different namespaces)